docs/leo-rover/documentation/specification.mdx

```diff
@@ -160,18 +162,27 @@ functionalities.
-| --------------------------------------- | ----------------------------------------------------------------- |
-| Power input                             | To power the board                                                |
-| RPi port                                | To power and RPi serial communication                             |
-| LED output                              | To control the battery LED (to show system readiness)             |
-| Motor output A, B, C & D (PWM H-bridge) | To power the rover motors and encodes                             |
-| 5-pin debug port                        | Used to flash firmware to the board using ST-link/V2 (optionally) |
+| Port                      | Functionality                                                                                                                                                                                                                                                                                 |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **J1** - Power Input      | <ul><li>DC 2.1/5.0 mm barrel jack</li><li>Reverse polarity protection</li><li>Overvoltage protection</li><li>10 A fast-blow fuse</li><li>Input voltage: 8–30 V DC</li></ul>                                                                                                                   |
+| **J2-J5** - Motor Outputs | <ul><li>4x PWM H-bridge motor outputs (A, B, C, D)</li><li>Peak current: ±2.8 A</li><li>Overcurrent (OCP) and short-circuit protection (to power and GND)</li><li>5 V encoder supply (max 1 A total for all motors)</li><li>2× encoder inputs per motor (active low – short to GND)</li></ul> |
+| **J7** - LED Output       | <ul><li>Controlled LED output (e.g. battery/system status)</li><li>Output: 5 V / 10 mA</li></ul>                                                                                                                                                                                              |
+| **JP4** - Fan Connector   | <ul><li>Controllable fan power output</li><li>Output: 5 V / 1 A</li></ul>                                                                                                                                                                                                                     |
+| **JP1** - UART Port       | <ul><li>UART communication interface</li><li>Logic level: 3.3 V</li></ul>                                                                                                                                                                                                                     |
```

Comment:
Ja bym to dał niżej i jakoś wspomniał, że to jest dodatkowe wyjście ten sam UART który jest na złączy JP3